.github/workflows/ci.yaml

```diff
@@ -107,6 +107,12 @@ jobs:
     - name: Test macaw-x86
       run: cabal test pkg:macaw-x86
 
+    - name: Build macaw-x86-syntax
+      run: cabal build pkg:macaw-x86-syntax
+
+    - name: Test macaw-x86-syntax
+      run: cabal test pkg:macaw-x86-syntax
+
```

Comment:
I think it would make more sense to run these jobs after the `macaw-x86-symbolic` ones, as `macaw-x86-syntax` depends on `macaw-x86-symbolic`.